### PR TITLE
feat(tooltip): allow children to be rendered instead of only text

### PR DESCRIPTION
**Pull request by fgameiro**
affects: @crave/farmblocks-tooltip

ISSUES CLOSED: #285

### CODE REVIEW

File: packages/tooltip/src/styledComponents/Tooltip.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const StyledTooltip = styled.div`
   box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.16);
   border: solid 1px rgba(0, 0, 0, 0.16);
   border-radius: 4px;
-  white-space: pre;
```
Comment (vNakamura) [inline packages/tooltip/src/styledComponents/Tooltip.js]: Why was this removed?

I don't know if it's necessary, but let's check. @fczuardi  do you remember why was this rule added?

File: packages/tooltip/src/Tooltip.story.js [hunk 2]
```diff
@@ -1,22 +1,37 @@
...
+        <Card>
+          this is a Card
+        </Card>
+      </Tooltip>
+    </div>
+  ))
+  .add("empty", () => (
+    <div style={{ position: "relative", width: "250px", border: "1px solid" }}>
+      <Tooltip isVisible={true} />
+    </div>
```
Comment (alcferreira) [inline packages/tooltip/src/Tooltip.story.js]: A `Tooltip` without content should not be rendered.

image

File: packages/tooltip/src/styledComponents/Tooltip.js (hunk 1 above)
Comment (fgameiro) [reply packages/tooltip/src/styledComponents/Tooltip.js]: This was preventing a long text from line breaking, but I think I can leave it on if this effect is expected.

Comment (fgameiro) [reply packages/tooltip/src/styledComponents/Tooltip.js]: With `white-space: pre;`
captura de tela de 2018-03-14 09-11-08

Without:
captura de tela de 2018-03-14 09-10-44

Comment (vNakamura) [reply packages/tooltip/src/styledComponents/Tooltip.js]: We talked about this, but to keep documented:
Let's keep the `white-space: pre;`
